engine/new_runtime/Cargo.toml

```
@@ -0,0 +1,24 @@
+[package]
+name = "stronghold-new_runtime"
+version = "0.1.0"
+edition = "2021"
+readme = "README.md"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[lib]
+name = "new_runtime"
+ 
+[dependencies]
+zeroize = { version = "1.4.3" }
+libsodium-sys = "0.2"
+serde = { version = "1.0", features = [ "derive" ] }
+random = { package = "rand", version = "0.8.4" }
+dirs = "4.0.0"
+
+[dependencies.iota-crypto]
+version = "0.8.0"
+features = [ "sha" ]
+
+[dev-dependencies]
+serde_json = "1.0"
```

```suggestion
serde_json = "1.0"

```
Nit: newline at end of file